Modules/LearningSequence/classes/Settings/class.ilLearningSequenceFilesystem.php

```diff
@@ -53,7 +67,7 @@ public function delete_image(string $which, ilLearningSequenceSettings $settings
             $delete = $settings->getExtroImage();
             $settings = $settings->withExtroImage();
         }
-
+        // TODO PHP8 Review: Undefined
```

Comment:
This exists in `ilFileSystemAbstractionStorage`, I wonder why PPHStorm complained here...